actions/entrypoint.sh

```diff
@@ -20,12 +20,19 @@ set -euo pipefail
 # GITHUB_SHA contains the commit hash.
 # GITHUB_WORKSPACE contains the repo folder.
 # GITHUB_EVENT_NAME contains the event name.
+# GITHUB_ACTIONS is true in GitHub env.
 
 export GITHUB_AUTH_TOKEN="$INPUT_REPO_TOKEN"
 export SCORECARD_V3=1
 export SCORECARD_POLICY_FILE="$INPUT_POLICY_FILE"
 export SCORECARD_SARIF_FILE="$INPUT_SARIF_FILE"
 
+if [[ "$GITHUB_EVENT_NAME" != "pull_request"* ]] && [[ "$GITHUB_REF" =~ ^refs/heads/(main|master)$ ]]; then
+    echo "$GITHUB_REF not supported with '$GITHUB_EVENT_NAME' event."
```

Comment:
According to `scorecard-analysis.yml`, GITHUB_EVENT_NAME="push" and GITHUB_REF="refs/heads/main" is supported right? 

But this would fail this check.

Reply:
you're right. Thanks for catching this mistake :-)
I updated. PTAL, I don't trust myself in the evening!